csharp-advanced-topics/DeprecationInCsharp/DeprecationInCsharp/StringUtils.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Globalization;
+
+namespace Deprecation
```

Comment:
We can use file scoped namespace here to reduce nesting.
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces